Multisig/UI/Transaction/Execution/FeeLegacyFormModel.swift

```diff
@@ -0,0 +1,195 @@
+//
+//  FeeLegacyFormModel.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 13.01.22.
+//  Copyright © 2022 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+import Solidity
+import Ethereum
+import UIKit
+
+class FeeLegacyFormModel: FormModel {
```

Comment:
why `legacy`?

Reply:
because there're 2 types of fees we'll work with: fee for a legacy-type EVM transaction, and a fee for an EIP-1559 type EVM transaction which will have different parameters